geary-papermc-features/src/main/kotlin/com/mineinabyss/geary/papermc/features/items/resourcepacks/ResourcePackGenerator.kt

```diff
@@ -57,6 +60,12 @@ class ResourcePackGenerator(world: Geary) : Geary by world {
                 .addTo(resourcePack)
         }
 
+        config.jarResources
+            .forEach {
+                val data = it.stream.use { Writable.copyInputStream(it) }
+                resourcePack.unknownFile(it.path, data)
+            }
+
```

Comment:
can likely inline it.stream here
personally would prefer checking the file-path and using the intended ResourcePack methods if its a model or not, might have cases where unknownFile causes issues with merging etc